mlflow/server/js/src/experiment-tracking/components/experiment-page/components/runs/cells/RunDescriptionCellRenderer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Tooltip } from '@databricks/design-system';
+import Utils from '../../../../../../common/utils/Utils';
+import { RunRowType } from '../../../utils/experimentPage.row-types';
+import { TrimmedText } from '../../../../../../common/components/TrimmedText';
+
+export const RunDescriptionCellRenderer = React.memo(({ value }: { value: RunRowType['tags'] }) => {
+  const description = Utils.getRunDescriptionFromTags(value) || '-';
+  return (
+    <>
+      <Tooltip title={description}>
+        <span>
+          <TrimmedText text={description} maxSize={50} className="model-name" />
```

Comment:
```suggestion
          <TrimmedText text={description} maxSize={50} />
```
+ I think you can safely make `className` prop optional

Reply:
![image](https://github.com/mlflow/mlflow/assets/103442620/e75a8a48-3494-422d-b6b3-eeef029dde48)
sir, it seems TrimmedText need className

Reply:
@zhouyou9505 you're free to make `className` in the `<TrimmedText>` an optional prop 🙂